sdk/highlight-ruby/highlight/lib/highlight.rb

```diff
@@ -86,13 +87,14 @@ def record_log(session_id, request_id, level, message, attrs = {})
         span.status = OpenTelemetry::Trace::Status.error(message) if [Logger::ERROR, Logger::FATAL].include?(level)
         span.add_event(LOG_EVENT, attributes: {
           LOG_SEVERITY_ATTRIBUTE => H.log_level_string(level),
-          LOG_MESSAGE_ATTRIBUTE => message,
+          LOG_MESSAGE_ATTRIBUTE => message.to_s,
           CODE_FILEPATH => caller_info[0],
           CODE_LINENO => caller_info[1],
           CODE_FUNCTION => function
         }.merge(attrs))
       end
     end
+    # rubocop:enable Metrics/AbcSize
```

Comment:
Unrelated, but I was getting [a CI failure](https://github.com/highlight/highlight/actions/runs/9275823170/job/25521424360) from Rubocop violations here that I don't think were introduced by my changes.